app/text_buffer.py

```diff
@@ -607,8 +610,12 @@ def find(self, searchFor, direction=0):
       self.doSelectionMode(app.selectable.kSelectionNone)
       return
     # The saved re is also used for highlighting.
-    self.findRe = re.compile('()'+searchFor)
-    self.findBackRe = re.compile('(.*)'+searchFor)
+    if self.ignoreCaseFind:
+      self.findRe = re.compile('()'+searchFor, re.IGNORECASE)
```

Comment:
wdyt of a flags variable set to something like

flags = self.ignoreCaseFind and re.IGNORECASE or 0
self.findRe = re.compile('()'+searchFor, flags)
self.findBackRe = re.compile('(.*)'+searchFor, flags)

and not having the if/else?